platform/linux-generic/odp_init.c

```diff
@@ -411,9 +408,6 @@ int odp_init_global(odp_instance_t *instance,
 	}
 	stage = TIMER_INIT;
 
-	/* No init neeeded */
-	stage = RANDOM_INIT;
```

Comment:
If this consecutive assignment is triggering some warnings, then how about commenting it out here but not touching the switch case in term_global()?